### DISCUSSION TITLE
Free upgrade to Enterprise Cloud

### DISCUSSION THREAD

**Discussion by jgrimme** [community/community#74830] [Enterprise]
### Select Topic Area

Question

### Body

we're looking to migrate our organization to an Enterprise Cloud account.  We have about 50 enterprise seats for the people in our 1 and only organization, but are looking to segment the user population into multiple organization, enable copilt, etc.

I've tried to contact sales with no luck, so I'm hoping the community will help me.  There is an option in the billing section of our organzations 'This account is eligible for a free upgrade to an enterprise account as part of Enterprise Cloud.'.  I was hoping to understand the implications of making that change/'upgrade', mostly in the context of if/how it will affect the members of the current organization.  Any insight would be appreciated.
thank you!

**Comment by sharon-upstream** [comment]
We will be glad to receive an answer to the same question as soon as possible.

Thanks!

**Comment by srikanth-tomo** [comment]
+1 Actually the copy writing and product marketing need to look into this. 
completely confusing. 

The banner asked to refer to this doc, it only confuses further. 
https://docs.github.com/en/enterprise-cloud@latest/admin/overview/about-enterprise-accounts

**Comment by npiasecki** [comment]
Basically it sounds nice but I'm scared to click the button because I'm not sure exactly what will happen and the docs don't really explain it

**Reply by drxddy** [reply]
same with me, would be helpful if someone can answer if they will start billing pricing after a free trial or after some certain period?

**Comment by isaacbowen** [comment]
I went for it. It seems fine so far! It created an enterprise account which now houses the original GitHub org. User access was unchanged. One of our builds broke because the default enterprise security policies don't allow GITHUB_TOKEN to have write access to the repo, but after fixing that it was fine.

This is not advice! I am not a lawyer and I cannot guarantee your experience in any way. :D

**Reply by drxddy** [reply]
Hey was that free or did you get an enterprise bill up until now, excited to share copilot with my team members : P

**Reply by isaacbowen** [reply]
Yes and no: the "enterprise" status seems to not have an associated charge of its own, but I am paying for à la carte enterprise features (namely GitHub Advanced Security).

To make it concrete, here's how that stuff shows up on our bill these days:
* GitHub Plan [x] seats ($21/month each)
* GitHub Copilot Business: [x] seats ($19/month each)
* GitHub Advanced Security [x] seats ($49/month each)

But yeah, no line items for the enterprise account itself.

**Reply by drxddy** [reply]
Thanks for the info, it confused me when as i saw it was already Enterprise in the billing section of my org.

**Comment by jamesmyers-blc** [comment]
One thing that I needed to also do was reapply GitHub Actions spending limits.  This seemed to have reset back to $0 which stopped pipeline jobs from running with this message "The job was not started because recent account payments have failed or your spending limit needs to be increased. Please check the 'Billing & plans' section in your settings."
Once I reapplied everything started to work again.

**Comment by npiasecki** [comment]
The messaging was getting more aggressive so I went ahead and clicked the button. Nothing broke as far as I can tell. Right now it seems more like a confusing extra layer unless you're a Fortune 500
